tests/CMakeLists.txt

```diff
@@ -1 +1,6 @@
 ADD_TEST(test ${PYTHON_EXECUTABLE} -m nose -s ${CMAKE_CURRENT_SOURCE_DIR})
+
+# For libdnf built with sanitizers, has no effect otherwise
```

Comment:
The comment is hard to understand, I think its missing "it" or something and possibly a period at the end of line?

Reply:
Yes, it is a separate sentence. I've added some punctuation to make it clearer.

Reply:
Thanks :+1: